File: hw1/src/test/java/CalculatorTest.java
import org.junit.Assert;
import org.junit.Test;

public class CalculatorTest {

    @Test
    public void calculatePlus() {
        int val1 = 2;
        int val2 = -4;
        String operator = "+";
        Assert.assertEquals(val1 + val2, Calculator.calculate(val1, operator, val2));
        Assert.assertEquals(0, Calculator.calculate(val2, "e", val1));
    }

    @Test
    public void calculateMinus() {
        int val1 = 2;
        int val2 = -4;
        String operator = "-";
        Assert.assertEquals(val1 - val2, Calculator.calculate(val1, operator, val2));
    }

    @Test
    public void calculateMultiplication() {
        int val1 = 2;
        int val2 = -4;
        String operator = "*";
        Assert.assertEquals(val1 * val2, Calculator.calculate(val1, operator, val2));
    }
    @Test
    public void calculateDivision() {
        int val1 = 2;
        int val2 = -4;
        String operator = "/";
        Assert.assertEquals(val2 / val1, Calculator.calculate(val2, operator, val1));
    }

}

File: hw2/src/test/CalculatorTest.kt
package test

import org.junit.Assert.*
import org.junit.Test

class CalculatorTest {
    @Test
    fun calculatePlus() {
        val val1 = 2
        val val2 = -4
        val operator = "+"
        assertEquals((val1 + val2).toLong(), null)
        assertEquals(0, null)
    }

    @Test
    fun calculateMinus() {
        val val1 = 2
        val val2 = -4
        val operator = "-"
        
       // assertEquals((val1 - val2).toLong(), Calculator.calculate(val1, operator, val2).toLong())
    }

    @Test
    fun calculateMultiplication() {
        val val1 = 2
        val val2 = -4
        val operator = "*"
       // assertEquals((val1 * val2).toLong(), Calculator.calculate(val1, operator, val2).toLong())
    }

    @Test
    fun calculateDivision() {
        val val1 = 2
        val val2 = -4
        val operator = "/"
       // assertEquals((val2 / val1).toLong(), Calculator.calculate(val2, operator, val1).toLong())
    }
}